Python.md

```diff
@@ -8,5 +8,9 @@
 
 1.  [Lesser Known Things about Python](https://github.com/jatinkatyal13/wtfpython)
 2.  [Jupyter Notebooks for Data Science Libraries of Python](https://github.com/nikhilkumarsingh/PythonForDataScience)
+3. [Python Faker Library to Generate fake Data](https://www.geeksforgeeks.org/python-faker-library/)
+4. [Socket Programming using Python](https://github.com/shauryauppal/Socket-Programming-Python)
```

Comment:
in socket programming url... You may provide the docs url or either mention that these are examples, in python and C.

Reply:
Example are in repo itself.

Readme is well maintain so that a newbie can also understand. All basics and basic examples like grocery store example code, CRC is mentioned in repi.

Reply:
I meant, write the heading like these are examples and not the official documentation.
**Socket Programming using Python Examples**